ydb/core/tx/schemeshard/schemeshard_info_types.h

```diff
@@ -2378,6 +2384,8 @@ struct TTableIndexInfo : public TSimpleRefCount<TTableIndexInfo> {
     TVector<TString> IndexDataColumns;
 
     TTableIndexInfo::TPtr AlterData = nullptr;
+
+    std::optional<NKikimrSchemeOp::TVectorIndexKmeansTreeDescription> VectorIndexKmeansTreeDescription;
```

Comment:
NIT: I don't think it's important now, but in general I think will be better something like this

```suggestion
    std::variant<std::monostate, NKikimrSchemeOp::TVectorIndexKmeansTreeDescription> SpecializedIndexDescription;
```

Reply:
You are right. Fixed.